convert string ressource dll: Invalid section name: CODE

I am using the latest Age2HD from Steam, with African Kingdoms expansion installed.

```
$ make run
./run game
INFO [py] No converted assets have been found
media conversion is required.
please provide your AoE II installation dir:
> /media/niklas/E4DE307EDE304AD6/Program Files (x86)/Steam/SteamApps/common/Age2HD
INFO [py] Game version(s) detected: Age of Empires 2: HD Edition (Version 3.x); Forgotten Empires; Age of Empires 2: The Conquerors, Patch 1.0c
INFO [py] converting metadata
INFO [py] [0] palette
INFO [py] [1] empires.dat
INFO [py] [2] blendomatic.dat
INFO [py] blending masks successfully exported
INFO [py] [3] player color palette
INFO [py] [4] terminal color palette
INFO [py] [5] string resources
Traceback (most recent call last):
  File "run.py", line 15, in init run (/home/niklas/Projekte/openage/run.cpp:832)
    main()
  File "/home/niklas/Projekte/openage/openage/__main__.py", line 94, in main
    return args.entrypoint(args, cli.error)
  File "/home/niklas/Projekte/openage/openage/game/main.py", line 33, in main
    if not convert_assets(assets, args):
  File "/home/niklas/Projekte/openage/openage/convert/main.py", line 125, in convert_assets
    for current_item in convert(args):
  File "/home/niklas/Projekte/openage/openage/convert/driver.py", line 126, in convert
    yield from convert_metadata(args)
  File "/home/niklas/Projekte/openage/openage/convert/driver.py", line 179, in convert_metadata
    stringres = get_string_resources(args)
  File "/home/niklas/Projekte/openage/openage/convert/driver.py", line 37, in get_string_resources
    pefile = PEFile(srcdir[name].open('rb'))
  File "/home/niklas/Projekte/openage/openage/convert/pefile.py", line 191, in __init__
    raise Exception("Invalid section name: " + section.name)
Exception: Invalid section name: CODE
Makefile:22: recipe for target 'run' failed
make: *** [run] Error 1
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/29733932-convert-string-ressource-dll-invalid-section-name-code?utm_campaign=plugin&utm_content=tracker%2F371436&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F371436&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/.mailmap
+++ b/.mailmap
@@ -10,3 +10,4 @@ Jonathan Remnant <jono4728@gmail.com>
 Henry Snoek <?> <snoek09@users.noreply.github.com>
 coop shell (Michael Enßlin, Jonas Jelten, Andre Kupka) <coop@sft.mx>
 Franz-Niclas Muschter <fm@stusta.net> <franz-niclas.muschter@stusta.net>
+Niklas Fiekas <niklas.fiekas@backscattering.de> <niklas.fiekas@tu-clausthal.de>

--- a/copying.md
+++ b/copying.md
@@ -49,7 +49,7 @@ _the openage authors_ are:
 | Jon Gelderloos              | jgelderloos                 | jgelderloos@gmail.com                 |
 | Emmanuel Gil Peyrot         | Link Mauve                  | linkmauve@linkmauve.fr                |
 | Danilo Bargen               | dbrgn                       | mail@dbrgn.ch                         |
-| Niklas Fiekas               | niklasf                     | niklas.fiekas@tu-clausthal.de         |
+| Niklas Fiekas               | niklasf                     | niklas.fiekas@backscattering.de       |
 | Charles Gould               | charlesrgould               | charles.r.gould@gmail.com             |
 | Wilco Kusee                 | detrumi                     | wilcokusee@gmail.com                  |
 | Sreejith R                  | sreejithr                   | sreejith.r44@gmail.com                |

--- a/openage/convert/driver.py
+++ b/openage/convert/driver.py
@@ -31,13 +31,7 @@ def get_string_resources(args):
 
     # AoK:TC uses .DLL PE files for its string resources,
     # HD uses plaintext files
-    if srcdir["language.dll"].is_file():
-        from .pefile import PEFile
-        for name in ["language.dll", "language_x1.dll", "language_x1_p1.dll"]:
-            pefile = PEFile(srcdir[name].open('rb'))
-            stringres.fill_from(pefile.resources().strings)
-            count += 1
-    elif GameVersion.age2_fe in args.game_versions:
+    if GameVersion.age2_fe in args.game_versions:
         from .hdlanguagefile import read_hd_language_file
 
         for lang in srcdir["resources"].list():
@@ -74,6 +68,12 @@ def get_string_resources(args):
                     # No utf-8 :(
                     stringres.fill_from(read_hd_language_file(langfile, lang, enc='iso-8859-1'))
                 count += 1
+    elif srcdir["language.dll"].is_file():
+        from .pefile import PEFile
+        for name in ["language.dll", "language_x1.dll", "language_x1_p1.dll"]:
+            pefile = PEFile(srcdir[name].open('rb'))
+            stringres.fill_from(pefile.resources().strings)
+            count += 1
 
     if not count:
         raise FileNotFoundError("could not find any language files")

